app/tests/groups/test_migrate_members_command.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management import call_command
+
+import pytest
+
+from app.content.factories import UserFactory
+from app.content.models import User
+from app.group.factories import GroupFactory
+from app.group.management.commands.migrate_members import GROUPS_TIHLDE_SLUG
+from app.group.models import Membership
+
+pytestmark = pytest.mark.django_db
+
+MIGRATE_COMMAND = "migrate_members"
+
+
+@pytest.fixture(autouse=True)
+def tihlde_group():
+    return GroupFactory(name=GROUPS_TIHLDE_SLUG)
+
+
+def test_migrate_groups_command_migrates_tihlde_members_to_tihlde_group():
+    """Test that all TIHLDE members are successfully migrated to the TIHLDE group."""
+    batch_size = 10
+    UserFactory.create_batch(batch_size, is_TIHLDE_member=True)
+
+    call_command(MIGRATE_COMMAND)
+
+    tihlde_members = User.objects.all()
+
+    assert (
+        user.membership.filter(group__slug=GROUPS_TIHLDE_SLUG).exists()
+        for user in tihlde_members
+    )
+    assert Membership.objects.count() == batch_size
```

Comment:
You are asserting more than the test implies. You are checking that users are migrated but also that no other memberships are created. Should this be it's own test?

Reply:
Good catch! Thanks for pointing it out 😄

Reply:
Good catch! Thanks for pointing it out 😄